README.md

```diff
@@ -163,6 +164,33 @@ let feedFromTwitter user pwd = taskSeq {
 }
 ```
 
+## Choosing between `AsyncSeq` and `TaskSeq`
+
+The [`AsyncSeq`][11] and `TaskSeq` library both operate on asynchronous sequences, but there are a few fundamental differences, most notably that the former _does not_ implement `IAsyncEnumerable<'T>`, but has its own same-named, but differently behaving type. Another core difference is that `TaskSeq` uses `ValueTasks` for the asynchronous computations, and `AsyncSeq` uses F#'s `Async<'T>`.
+
+There are more differences:
+
+|                            | `TaskSeq`                                                                             | `AsyncSeq`                                                           |
+|----------------------------|---------------------------------------------------------------------------------------|----------------------------------------------------------------------|
+| **Frameworks**             | .NET 5.0+, NetStandard 2.1                                                        | .NET 5.0+, NetStandard 2.0 and 2.1, .NET Framework 4.6.1+            |
+| **Underlying type**        | `System.Collections.Generic.IAsyncEnumerable<'T>`                                 | Its own type, also called `IAsyncEnumerable<'T>`, but not compatible |
+| **Implementation**         | State machine (statically compiled)                                               | No state machine, continuation style                                 |
+| **Semantics**              | `seq`-like: on-demand                                                             | `seq`-like: on-demand                                                |
+| **Support `let!`**         | All `task`-like: `Async<'T>`, `ValueTask<'T>`, `Task<'T>` or any `GetAwaiter()`   | `Async<'T>` only                                                     |
+| **Support `do!`**          | `Async<unit>`, `Task<unit>` and `Task`, `ValueTask<unit>` and `ValueTask`         | `Async<unit>` only                                                   |
```

Comment:
```suggestion
| **Support `do!`**          | `Async<unit>`, `Task<unit>`, `Task`, `ValueTask<unit>`, `ValueTask`         | `Async<unit>` only                                                   |
```

Reply:
I like to keep this one, as it is deliberate (I had commas before but didn't like it), it signifies they're interchangeable semantically.